charts/kube-prometheus-stack/templates/alertmanager/servicemonitor.yaml

```diff
@@ -36,17 +36,15 @@ spec:
     bearerTokenFile: {{ .Values.alertmanager.serviceMonitor.bearerTokenFile }}
     {{- end }}
     {{- if .Values.alertmanager.serviceMonitor.tlsConfig }}
-    tlsConfig: {{ toYaml .Values.alertmanager.serviceMonitor.tlsConfig | nindent 6 }}
+    tlsConfig: {{- toYaml .Values.alertmanager.serviceMonitor.tlsConfig | nindent 6 }}
     {{- end }}
     path: "{{ trimSuffix "/" .Values.alertmanager.alertmanagerSpec.routePrefix }}/metrics"
-{{- if .Values.alertmanager.serviceMonitor.metricRelabelings }}
-    metricRelabelings:
-{{ tpl (toYaml .Values.alertmanager.serviceMonitor.metricRelabelings | indent 6) . }}
-{{- end }}
-{{- if .Values.alertmanager.serviceMonitor.relabelings }}
-    relabelings:
-{{ toYaml .Values.alertmanager.serviceMonitor.relabelings | indent 6 }}
-{{- end }}
+    {{- if .Values.alertmanager.serviceMonitor.metricRelabelings }}
+    metricRelabelings: {{- tpl (toYaml .Values.alertmanager.serviceMonitor.metricRelabelings | nindent 6) . }}
```

Comment:
Why would this even require nindent?

Reply:
My purpose in changing this was just to make the code more readable.  The outcome should be identical.  I've just run a `helm template` on my branch as well the main branch (with a values file that defines both relabelings and metricsRelabelings) and and the outcomes are the identical.

The [documentation](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint) confirms both are expected to be a `[]RelabelConfig` list, so nindent sounds right to me.